src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,31 +1,32 @@
 import { Search } from "react-bootstrap-icons";
 import { useState } from "react";
-import AsyncSelect from "react-select/async";
-
+import { AsyncPaginate } from "react-select-async-paginate";
 import fetchCities from "../../api/fetchCities";
 import { SearchDiv, SearchIconContainer, selectStyles } from "./styled";
 
 const SearchBar = () => {
   const [error, setError] = useState(null);
 
-  const loadOptions = async (inputValue, callback) => {
+  const loadOptions = async (inputValue) => {
     if (inputValue !== "") {
       try {
         const result = await fetchCities(inputValue);
         setError(null);
-        callback(result);
+        return result;
       } catch (e) {
         setError(e);
+        return null;
```

Comment:
why return null? Maybe return { options: [] };? And you able to combine these cases